manifests/addons/s3-blobstore.yml

```diff
@@ -0,0 +1,25 @@
+---
+- type: remove
+  path: /instance_groups/name=bosh/jobs/name=blobstore
+
+- type: replace
+  path: /instance_groups/name=bosh/properties/blobstore?
+  value:
+    provider: s3
+    access_key_id: (( vault meta.vault "/aws/blobstore:access_key" ))
+    secret_access_key: (( vault meta.vault "/aws/blobstore:secret_key" ))
+    bucket_name: (( vault meta.vault "/aws/blobstore:bucket" ))
+    s3_region: (( vault meta.vault "/aws/blobstore:region" ))
```

Comment:
bucket_name and s3_region aren't really secrets, persay, and should be moved out of the vault into params.